src/utils/number.cljs

```diff
@@ -29,3 +30,13 @@
   if `num` exceeds a given bound, then returns the bound exceeded."
   [number lower-bound upper-bound]
   (max lower-bound (min number upper-bound)))
+
+(defn remove-trailing-zeroes
```

Comment:
probably should add a test to cover this functionality? 🤔 

is this really core "numbers" or is it more specific to the wallet also? 🤔

Reply:
Well, this function was already in a common file for wallet, but moved to a utils as it is a general purpose function and needed to use it un token input component

Reply:
Good point 😅